app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+def channels
+  channels = Channel.all.map { |channel| channel.id }
+  data = JSON.pretty_generate(channels)
+
+  render(json: data)
+
+end
+
+def channels_uuid
+
+  channel = Channel.find_by_id(params[:uuid])
+
+  case channel.details_type
+  when "SiteChannelDetails"
+    channel_details = SiteChannelDetails.find_by_id(channel.details_id)
+    platform = "Site"
+    channel_id = channel_details.brave_publisher_id
+  when "YoutubeChannelDetails"
+    channel_details = YoutubeChannelDetails.find_by_id(channel.details_id)
+    platform = "Youtube"
+    channel_id = channel_details.youtube_channel_id
+  when "TwitterChannelDetails"
+    channel_details = TwitterChannelDetails.find_by_id(channel.details_id)
+    platform = "Twitter"
+    channel_id = channel_details.twitter_channel_id
+  when "TwitchChannelDetails"
+    channel_details = TwitchChannelDetails.find_by_id(channel.details_id)
+    platform = "Twitch"
+    channel_id = channel_details.twitch_channel_id
+  end
+
+  data = JSON.pretty_generate({
+    uuid: channel.id,
+    platform: platform,
+    name: channel_details.name,
+    channel_id: channel_id,
+    stats: channel_details.stats,
+    publisher_id: channel.publisher_id,
```

Comment:
I think we want `channel.publisher.owner_identifier` here:
```
irb(main):121:0> channel.publisher_id
=> "08df5822-557c-4944-a38e-ef8c7b557f96"
irb(main):122:0> channel.publisher.owner_identifier
=> "publishers#uuid:08df5822-557c-4944-a38e-ef8c7b557f96"
```

Reply:
also prefer `owner_id` for consistency and other past naming convention problems https://github.com/brave-intl/publishers/blob/8b6093cb8fc8487534e1e898d0a224ef00a0166c/app/services/promo_registrar.rb#L116

Reply:
Fixed